app/views/foods/index.erb

```diff
@@ -3,5 +3,7 @@
 <div class="foods">
   <% @foods.each do |food| %>
     <h3 class="name"><%= food.name %></h3>
+    <li>
```

Comment:
we need a closing tag here

Reply:
@Tresor11  I correct everything already , you can now check :+1: